gensrc/thrift/AgentService.thrift

```diff
@@ -395,6 +395,8 @@ struct TPublishVersionRequest {
     2: required list<TPartitionVersionInfo> partition_version_infos
     // strict mode means BE will check tablet missing version
     3: optional bool strict_mode = false
+    // for delta rows statistics to exclude rollup tablets
+    4: optional set<Types.TTabletId> base_tablet_ids
```

Comment:
could we use index id instead to save FE memory and RPC data transfer?

Reply:
Yes, I'll get it done.

Reply:
<img width="1069" alt="image" src="https://github.com/apache/doris/assets/82279870/afc680df-4c09-46ca-bc62-4cc84dc41a43">
However TabletMeta::index_id was added 2 months ago, afraid that change won't take effect on the elder tables.
But the original way won't have that problem.